at_onboarding_flutter/pubspec.yaml

```diff
@@ -11,19 +11,49 @@ environment:
 dependencies:
   flutter:
     sdk: flutter
-  archive: ^2.0.13
-  file_picker: ^2.0.7
-  flutter_qr_reader: ^1.0.3
-  url_launcher: ^5.7.0
-  permission_handler: ^5.0.1+1
-  path_provider: ^1.6.24
-  webview_flutter: ^1.0.7
-  at_utils: ^1.0.1+6
-  at_backupkey_flutter: ^1.0.0+2
-  at_server_status: ^0.1.0+1
-  at_client_mobile: ^1.0.0+9
-  at_commons: ^1.0.1+6
-  at_client: ^1.0.1+8
+  archive: ^3.1.2
+  file_picker: ^3.0.1
+  flutter_qr_reader: ^1.0.5
+  url_launcher: ^6.0.3
+  permission_handler: ^6.1.3
+  path_provider: ^2.0.1
+  webview_flutter: ^2.0.4
+  at_utils:
+    git:
+      url: https://github.com/atsign-foundation/at_tools.git
+      path: at_utils
+      ref: dev_env
```

Comment:
have to change git ref to pub spec version

Reply:
sure